### ISSUE TITLE
Twitch Activity Dock does not load content (anymore)

### ISSUE THREAD

**Issue by mihawk90** [obsproject/obs-browser#334]
### Operating System Info

Other

### Other OS

Fedora 33

### OBS Studio Version

27.1.3

### OBS Studio Log URL

https://obsproject.com/logs/aHChhyg8MgHQom3O

### OBS Studio Crash Log URL

---

### Expected Behavior

Twitch Activities showing in the respective dock.

### Current Behavior

Since today, the Twitch Activity dock does not show content anymore.

### Steps to Reproduce

1. Open OBS with a connected Twitch account
2. Show Twitch Activity dock
3. See it load black with no content

### Anything else we should know?

The issue is not on OBS' side, there was a change in the URL Twitch uses to access the feed, so you'd have to check with your Twich contacts.

It is still possible to add a working Activity dock using a custom dock, however the URL now contains a uuid, which I don't know how it is generated or where it's from:
```
https://dashboard.twitch.tv/popout/u/tarulia/stream-manager/activity-feed?uuid=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
```

When removing the uuid, it also does not load in the browser. However, even with a fake UUID (literally the link above with the x's), it loads perfectly fine... so it doesn't look like it's actually relevant?

**Comment by WizardCM** [comment]
I did some testing earlier and confirmed what you've mentioned. Essentially, they released a v2 of the Activity Feed with a better way of saving your selected filters, and for some reason they persist it using an auto-generated UUID (which seems to be tied to the login session). If you use `localStorage.getItem('dashboardMultiActivityFeedV2Filters')` you'll see the UUID as the key for the filters.

I have passed it on internally and we'll see what happens - it's possible we'll have to generate a UUID ourselves.
